exhibitor_tls_artifacts/gen_artifacts.py

```diff
@@ -1,44 +1,68 @@
 import click
 import os
 import shutil
+import sys
+from pathlib import Path
 
 from .gen_certificates import CertificateGenerator
 from .gen_stores import KeystoreGenerator
 
 
 @click.command()
-@click.argument('sans', nargs=-1)
+@click.argument('nodes', nargs=-1)
 @click.option('-d', '--dir', help='Directory to put artifacts in.',
               default='./artifacts/')
-def app(sans, dir):
+def app(nodes, dir):
+    """
+    Generates Admin Router and Exhibitor TLS artifacts. NODES should consist
+    of a space seperated list of master ip addresses. See
+    https://docs.mesosphere.com/1.13/security/ent/tls-ssl/exhibitor-tls/
+    """
+    dir = Path(dir)
```

Comment:
Optional because this is an existing problem, but [`dir`](https://docs.python.org/3/library/functions.html#dir) is a Python builtin and so we should not use it.

Reply:
👍